spawn_operator/spawn_operator/lib/spawn_operator/k8s/proxy/deployment.ex

```diff
@@ -203,7 +203,7 @@ defmodule SpawnOperator.K8s.Proxy.Deployment do
     proxy_container = %{
       "name" => "spawn-sidecar",
       "image" => "#{annotations.proxy_image_tag}",
-      "env" => @default_actor_host_function_env,
+      "env" => actor_host_function_envs,
```

Comment:
I really don't like it. The user should not configure the proxy here. It should use Operator's annotations mechanism to do any configuration to the proxy. I know it's different than the way we do it with the user container, but that was on purpose. So that we always have control of what can or cannot be done with the proxy in terms of configuration.